mpas_analysis/config.default

```diff
@@ -1137,40 +1161,6 @@ normArgsDifference = {'vmin': -0.3, 'vmax': 0.3}
 # colorbarTicksDifference = numpy.linspace(-0.3, 0.3, 9)
 
 
-[transects]
```

Comment:
There's a bunch more stuff here that also got deleted. Again, probably something went wrong with a merge?